doc/classes/SplitContainer.xml

```diff
@@ -42,30 +79,37 @@
 	</signals>
 	<constants>
 		<constant name="DRAGGER_VISIBLE" value="0" enum="DraggerVisibility">
-			The split dragger is visible when the cursor hovers it.
+			The split dragger icon is always visible when [theme_item autohide] is [code]false[/code], otherwise visible only when the cursor hovers it.
+			The dragger icon is automatically hidden if the length of the grabber icon is longer than the split bar.
 		</constant>
 		<constant name="DRAGGER_HIDDEN" value="1" enum="DraggerVisibility">
-			The split dragger is never visible.
+			The split dragger icon is never visible.
 		</constant>
 		<constant name="DRAGGER_HIDDEN_COLLAPSED" value="2" enum="DraggerVisibility">
-			The split dragger is never visible and its space collapsed.
+			The split dragger icon is not visible and the split bar is collapsed to zero width.
+		</constant>
+		<constant name="DRAGGER_HIDDEN_USE_SEPARATION" value="3" enum="DraggerVisibility">
```

Comment:
This seems to be the same behavior as `DRAGGER_HIDDEN` with no icon. Isn't it redundant?

Reply:
I mean `DRAGGER_HIDDEN` makes the icon never visible, so you can as well set it to null.

Reply:
So are you advising to go ahead and remove the `DRAGGER_HIDDEN` option?

Reply:
No. `DRAGGER_HIDDEN` makes the icon invisible, but still uses its size. But if you remove the icon, it will use separation.
`DRAGGER_HIDDEN_USE_SEPARATION` makes the icon invisible and uses separation.
So what I'm saying is, if the icon is invisible anyway, why can't you just remove it if you want to use separation? Which makes the `DRAGGER_HIDDEN_USE_SEPARATION` redundant. Unless I'm missing something?

Reply:
Ah, I didn't really test this, I though the icon can be null 🤔
In that case a new dragger mode *probably* makes sense, but it feels like it's circumventing a Control limiation.
Empty ImageTexture does not seem to cause any errors. Isn't it a viable workaround?

Reply:
I will defer to your opinion regarding the engine's philosophy on whether a workaround is okay under certain conditions. The workaround could be put in the docs in the visibility settings.

Can you elaborate on "circumventing a Control limitation"?

A knock-on benefit of the `grabber_sets_minimum_separation` idea, is that you could then do some interesting things with for example, an image that is cropped by the separation area, because it remains behind the child Controls. Not a reason enough on its own, but adds some flexibility to the split container.

Reply:
> Can you elaborate on "circumventing a Control limitation"?

Controls don't support null icons in theme properties. Although I checked the default theme and empty ImageTexture seems to be used as "empty icon", so is somewhat official way to remove icons.

> I will defer to your opinion regarding the engine's philosophy on whether a workaround is okay under certain conditions. The workaround could be put in the docs in the visibility settings.

IMO using SplitContainer without icon is kind of an edge case that I didn't see a proposal for. So given what I said above about empty ImageTexture, I think it's fine if it's mentioned in the documentation.

Reply:
Okay. So to summarize, I will remove `DRAGGER_HIDDEN_USE_SEPARATION`, and update the docs to include the empty `ImageTexture` workaround for `0` thickness icon?

Reply:
Yes.
`DRAGGER_HIDDEN_USE_SEPARATION` is something that can be easily added later if there is a demand, but for now there doesn't seem to be a reason to do so.